ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -1035,10 +1462,13 @@ def _addToSolutionExport(self, traj, rlz, acceptable):
     # meta variables
     toAdd = {'age': 0 if self.popAge is None else self.popAge,
              'batchId': self.batchId,
-             'fitness': rlz['fitness'],
              'AHDp': self.ahdp,
              'AHD': self.ahd,
-             'HDSM': self.hdsm}
+             'rank': 0 if not self._isMultiObjective else rlz['rank'],
+             'CD': 0 if not self._isMultiObjective else rlz['CD'],
```

Comment:
I would suggest we set default values in rlz['rank'] = 0 and rzl['CD'] = 0, in this case, we will have less distinction between single and multi-objective cases. .

Reply:
Rank and crowding distance are not particularly meaningful in single objective, so I just removed them from the solution export  in 1cad17e3399597bcfa9fa5dd97fa27716b603192